testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/util/TemporaryHiveTable.java

```diff
@@ -13,31 +13,17 @@
  */
 package io.trino.tests.product.hive.util;
 
-import java.security.SecureRandom;
-
 import static io.trino.tests.product.utils.QueryExecutors.onHive;
-import static java.lang.Character.MAX_RADIX;
-import static java.lang.Math.abs;
-import static java.lang.Math.min;
 import static java.util.Objects.requireNonNull;
 
 public class TemporaryHiveTable
         implements AutoCloseable
 {
-    private static final SecureRandom random = new SecureRandom();
-    private static final int RANDOM_SUFFIX_LENGTH = 12;
```

Comment:
I wonder how much time were we wasting here. 😄

Reply:
the new impl is for sure slower